kiwi/builder/disk.py

```diff
@@ -206,8 +206,15 @@ def __init__(self, xml_state, target_dir, root_dir):
         self.disk_setup = DiskSetup(
             xml_state, root_dir
         )
+        self.custom_args = custom_args
+
+        self.boot_signing_keys = None
+        if custom_args and 'signing_keys' in custom_args:
+            self.boot_signing_keys = custom_args['signing_keys']
+
         self.boot_image = BootImage(
-            xml_state, target_dir, root_dir
+            xml_state, target_dir, root_dir,
+            signing_keys=self.boot_signing_keys
```

Comment:
same here key=value style seems to provide redundant information
I think the name boot_signing_keys is misleading, sounds like we provide a key
for signing the boot code

```python
self.boot_image = BootImage(
    xml_state, target_dir, root_dir, self.boot_image_signing_keys
)
```